CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # CHANGELOG
 
+## 2.5.0 (2023-07-13)
+
+MLflow 2.5.0 includes several major features and improvements
+
+Features:
+
+- [] MLflow Gateway (#8694, @harupy)
```

Comment:
This entry definitely needs more words or deserves its own section.

Reply:
It should be a top-level announcement (banner) as we have done in the past with major releases.

Reply:
Perhaps not as wordy as https://github.com/mlflow/mlflow/releases/tag/v2.0.1 , but something similar that talks about the fact that this is an entire new direction that MLflow is expanding into with the AI Gateway Server :)